yarn-project/circuit-types/src/tx/tx.ts

```diff
@@ -47,8 +52,13 @@ export class Tx extends Gossipable {
     public encryptedLogs: EncryptedTxL2Logs,
     /**
      * Unencrypted logs generated by the tx.
+     * NOTE: These do not get filled, but remain here so enqueued_calls_processor.ts can accumulate public logs
```

Comment:
We really shouldn't change the tx directly for new logs :( 
But I'll refactor it in my next pr!